lib/ansible/cli/__init__.py

```diff
@@ -575,6 +575,13 @@ def parse(self):
 
         self.options, self.args = self.parser.parse_args(self.args[1:])
 
+        # --limit must receive non-zero length string
+        if hasattr(self.options, 'subset') and  \
+           self.options.subset != C.DEFAULT_SUBSET and \
+           isinstance(self.options.subset, string_types) and \
+           len(self.options.subset) == 0:
```

Comment:
Instead of indenting these lines to align with `hasattr` indent them 1 full indentation further than the next line.

```
        if hasattr(self.options, 'subset') and  \
                self.options.subset != C.DEFAULT_SUBSET and \
                isinstance(self.options.subset, string_types) and \
                len(self.options.subset) == 0:
            raise ...
```

Reply:
it's done